spdmlib/src/common/mod.rs

```diff
@@ -1154,12 +1459,16 @@ bitflags! {
 pub struct SpdmRuntimeInfo {
     connection_state: SpdmConnectionState,
     last_session_id: Option<u32>,
+    local_used_cert_chain_slot_id: u8,
+    peer_used_cert_chain_slot_id: u8,
     pub need_measurement_summary_hash: bool,
     pub need_measurement_signature: bool,
     pub message_a: ManagedBufferA,
     pub message_b: ManagedBufferB,
     pub message_c: ManagedBufferC,
     pub message_m: ManagedBufferM,
+    pub message_mut_b: ManagedBufferB,
+    pub message_mut_c: ManagedBufferC,
```

Comment:
`message_mut_b` and `message_mut_c` are for basic mutual auth, which is deprecated. Why we need them?